### ISSUE TITLE
Help using and installing this 

### ISSUE THREAD

**Issue by Deelowz** [marcelhoffs/input-switcher#16]
I'm a very new user of github repos and I have no idea how to install or use this program. I tried using the README file but I have no clue. I really need the inputswitcher to work for my mouse and keyboard. Please assist.

**Comment by phelep17** [comment]
Agreed, it is not easy to understand and follow the steps.. some points are not obvious.
